### DISCUSSION TITLE
please help me

### DISCUSSION THREAD

**Discussion by GentleKnight-02** [community/community#135665] [Programming Help] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

 const tx = await program.methods.initialize().accounts({counter:counter.publicKey,user:provider.wallet.publicKey}).signers([counter]).rpc();
    console.log("Your transaction signature", tx);
    
  but i can't send this transaction flow this error.

  can not get block hash.

### Guidelines

- [ ] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by apartyagi** [comment]
hlo
please explain your problem i will help you

**Reply by GentleKnight-02** [reply]
> hlo please explain your problem i will help you

I update my issue.

**Comment by 0xp3p3x0** [comment]
did you get provider from anchor?

**Comment by 0xp3p3x0** [comment, marked as answer]
like this 
const provider = anchor.AnchorProvider.env();
  anchor.setProvider(anchor.AnchorProvider.env());

**Reply by GentleKnight-02** [reply]
thank,sir

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @GentleKnight-02 ! 

We're happy you're here. You are more likely to get a useful response if you include as many relevant details as you can and are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
